### PR TITLE
Support sum and mean for Stackdriver exporter

### PR DESCRIPTION
**Pull request by songy23**
Fix https://github.com/census-instrumentation/opencensus-go/issues/180

### CODE REVIEW

File: exporter/stats/stackdriver/stackdriver_test.go [hunk 1]
```diff
@@ -331,9 +470,7 @@ func TestEqualAggWindowTagKeys(t *testing.T) {
 	}
 }
 
-func newTestCumViewData(v *stats.View, start, end time.Time) *stats.ViewData {
-	count1 := stats.CountData(10)
-	count2 := stats.CountData(16)
+func newTestCumViewData(v *stats.View, start, end time.Time, data1 stats.AggregationData, data2 stats.AggregationData) *stats.ViewData {
```
Comment (odeke-em) [inline exporter/stats/stackdriver/stackdriver_test.go]: Minor nit `data1, data2 stats.AggregationData)`

Comment (songy23) [reply exporter/stats/stackdriver/stackdriver_test.go]: Sure, updated.

File: exporter/stats/stackdriver/stackdriver.go [hunk 2]
```diff
@@ -299,6 +303,14 @@ func newTypedValue(view *stats.View, r *stats.Row) *monitoringpb.TypedValue {
 		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_Int64Value{
 			Int64Value: int64(*v),
 		}}
+	case *stats.SumData:
+		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_DoubleValue{
+			DoubleValue: float64(*v),
+		}}
+	case *stats.MeanData:
+		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_DoubleValue{
+			DoubleValue: v.Mean,
```
Comment (jcd2) [inline exporter/stats/stackdriver/stackdriver.go]: A cumulative metric containing just the mean isn't going to work very well.  I suggest converting the MeanData to a distribution.  (You could also use a gauge metric, but aggregating across timeseries wouldn't work as well.)

File: exporter/stats/stackdriver/stackdriver.go [hunk 3]
```diff
@@ -242,6 +242,10 @@ func (e *Exporter) createMeasure(ctx context.Context, vd *stats.ViewData) error
 	switch agg.(type) {
 	case stats.CountAggregation:
 		valueType = metricpb.MetricDescriptor_INT64
+	case stats.SumAggregation:
```
Comment (jcd2) [inline exporter/stats/stackdriver/stackdriver.go]: This can't work without updating equalAggWindowTagKeys.

Comment (songy23) [reply exporter/stats/stackdriver/stackdriver.go]: Thanks for catching this, I updated this in https://github.com/census-instrumentation/opencensus-go/pull/224

File: exporter/stats/stackdriver/stackdriver.go (hunk 2 above)
Comment (songy23) [reply exporter/stats/stackdriver/stackdriver.go]: Make sense to me. Also updated in https://github.com/census-instrumentation/opencensus-go/pull/224

**Pull request merged**